From f6058d38ef95f260889be2bfef3ee639f4366ac8 Mon Sep 17 00:00:00 2001
From: Carles Fernandez <carles.fernandez@gmail.com>
Date: Sun, 6 Sep 2020 11:34:10 +0200
Subject: [PATCH] Remove kernel from blacklist since it is already patched

---
 blacklist/kprobes.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/blacklist/kprobes.h b/blacklist/kprobes.h
index 6c1d7a5..cfc4160 100644
--- a/blacklist/kprobes.h
+++ b/blacklist/kprobes.h
@@ -13,8 +13,7 @@
 #include <lttng-kernel-version.h>
 
 #if LTTNG_KERNEL_RANGE(4,20,0, 4,20,13) \
-       || LTTNG_KERNEL_RANGE(4,19,9, 4,19,26) \
-       || LTTNG_KERNEL_RANGE(4,14,87, 4,14,104)
+       || LTTNG_KERNEL_RANGE(4,19,9, 4,19,26)
 # if defined(CONFIG_ARM) && defined(CONFIG_OPTPROBES)
 #  error "Your kernel is known to have buggy optimized kprobes implementation. Fixed by commit 0ac569bf6a7983c0c5747d6df8db9dc05bc92b6c \"ARM: 8834/1: Fix: kprobes: optimized kprobes illegal instruction\" in Linu$
 # endif /* #if defined(CONFIG_ARM) && defined(CONFIG_OPTPROBES) */

