From 83ad897e626ed2f165246e38caca9d831e0f933f Mon Sep 17 00:00:00 2001
From: Carles Fernandez <carles.fernandez@gmail.com>
Date: Mon, 31 Aug 2020 16:50:34 +0200
Subject: [PATCH] Remove examples

---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index ac1c11f..c7e1818 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -3,7 +3,7 @@
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtkdatabox.pc

-SUBDIRS = gtk glade examples
+SUBDIRS = gtk glade

 EXTRA_DIST = depcomp scripts gtkdatabox.pc.in gtkdatabox.spec.in glade/gtkdatabox.xml api
