--- hdf5-1.8.15-patch1/configure.ac.orig	2015-09-26 20:57:43.764764359 +0100
+++ hdf5-1.8.15-patch1/configure.ac	2015-09-26 20:58:11.021955389 +0100
@@ -1719,24 +1719,7 @@
 ## Check if pthread_attr_setscope(&attribute, PTHREAD_SCOPE_SYSTEM)
 ## is supported on this system
 ##
-AC_MSG_CHECKING([Threads support system scope])
-AC_CACHE_VAL([hdf5_cv_system_scope_threads],
-  [AC_TRY_RUN([
-    #if STDC_HEADERS
-    #include <stdlib.h>
-    #include <pthread.h>
-    #endif
-
-    int main(void)
-    {
-      pthread_attr_t attribute;
-      int ret;
-
-      pthread_attr_init(&attribute);
-      ret=pthread_attr_setscope(&attribute, PTHREAD_SCOPE_SYSTEM);
-      exit(ret==0 ? 0 : 1);
-    }
-  ], [hdf5_cv_system_scope_threads=yes], [hdf5_cv_system_scope_threads=no],)])
+hdf5_cv_system_scope_threads=no
 
 if test ${hdf5_cv_system_scope_threads} = "yes"; then
   AC_DEFINE([SYSTEM_SCOPE_THREADS], [1],
